cdap-hbase-compat-base/src/main/java/co/cask/cdap/data2/util/hbase/HBaseVersion.java

```diff
@@ -123,19 +123,26 @@ public static void main(String[] args) {
 
   /**
    * Utility class to parse apart version number components.  The version string provided is expected to be in
-   * the format: major[.minor[.patch[.last]][-classifier][-SNAPSHOT]
+   * one of the formats below.
    *
-   * <p>Only the major version number is actually required.</p>
+   * <p>Only the major version number and classifier are actually required.</p>
    */
   public static class VersionNumber {
+    // major[.minor[.patch[.last]][-classifier][-SNAPSHOT]
     private static final Pattern PATTERN =
```

Comment:
Would be good to add a comment to say what this pattern is used for. If this is used only for CDH, can we call this `CDH_PATTERN`?